modules/sso-rbac/krolebinding.tf

```diff
@@ -0,0 +1,21 @@
+resource "kubernetes_role_binding" "example" {
```

Comment:
please use some meaningful names for resource/module names or just "this" instead of "example"
also please have module/resource names with snake_case casing